deploy/Dockerfile

```diff
@@ -17,7 +17,7 @@
 # how you want, search for "ENV AGENT_" in this file.
 
 # Stage 1: Set python image as base image
-FROM python:3.12-slim AS builder
+FROM python:3.13-slim AS builder
```

Comment:
Using python 3.13 by default now, as it's probably what most people are going to use from now on